src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -826,7 +826,7 @@ public async Task TestOpenProject_AssemblyNameIsPath()
                 var project = solution.Projects.First();
                 var comp = await project.GetCompilationAsync();
                 Assert.Equal("ReproApp", comp.AssemblyName);
-                var expectedOutputPath = GetParentDirOfParentDirOfContainingDir(project.FilePath);
+                var expectedOutputPath = Path.GetDirectoryName(project.FilePath);
```

Comment:
Looks like GetParentDirOfParentDirOfContainingDir can be deleted now?

Reply:
done